skimage/feature/peak.py

```diff
@@ -11,8 +11,8 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     Peaks are the local maxima in a region of `2 * min_distance + 1`
     (i.e. peaks are separated by at least `min_distance`).
 
-    If peaks are flat (i.e. multiple adjacent pixels have identical
-    intensities), the coordinates of all such pixels are returned.
+    If multiple peaks in the specified region have identical intensities,
+    the coordinates of all such pixels are returned.
```

Comment:
@soupault @JDWarner I agree that there's no *easy* way to break ties, but breaking ties is almost certainly what users would want (hence why the issues were raised in the first place). You could go by size of the connected component, or if that's also equal, toss a coin or something. But the point is that this would often be used to e.g. seed a watershed, and the idea of passing a min_distance is to minimize oversegmentation, ie limit the number of very-close-together seeds. As it stands, I don't really see a utility for the kwarg...